web-components/src/components/usa-identifier/identifier.stories.js

```diff
@@ -0,0 +1,154 @@
+import "./index";
+import { html } from 'lit';
+
+export default {
+  title: "Components/Identifier",
+  component: "usa-identifier",
+  argTypes: {
+    primary_agency: { name: "Primary agency information" },
+    logo1: { control: 'text', name: "Primary agency logo" },
+    secondary_agency: {name: "Secondary agency information" },
+    logo2: { control: 'text', name: "Secondary agency logo" },
+    masthead: { name: "Masthead content" },
+    taxpayer: { name: "Taxpayer disclaimer" },
+    required_links: { name: "Required links" },
+    usagov: { name: "USA.gov information" }
+  },
+  args: {
+    primary_agency: {
+      name: "[Parent agency]",
+      shortname: "[Agency shortname]",
+      url: "javascipt:void(0)",
+    },
+    logo1: "https://designsystem.digital.gov/assets/img/circle-gray-20.svg",
+    secondary_agency: {
+      name: "[Other agency]",
+      url: "javascipt:void(0)",
+    },
+    logo2: "https://designsystem.digital.gov/assets/img/circle-gray-20.svg",
+    masthead: {
+      conjunction: "and",
+      domain: "[domain.gov]",
+      disclaimer: "An official website of the",
+    },
+    taxpayer: "Produced and published at taxpayer expense.",
+    required_links: {
+      about: "About",
+      accessibility: "Accessibility statement",
+      foia: "FOIA requests",
+      no_FEAR: "No FEAR Act data",
+      oig: "Office of the Inspector General",
+      performance: "Performance reports",
+      privacy: "Privacy policy",
+    },
+    usagov: {
+      text: "Looking for U.S. government information and services?",
+      link_label: "Visit USA.gov",
+      link_url: "https://www.usa.gov/"
+    }
+  },
+  render: ({
+    primary_agency,
+    secondary_agency,
+    logo1,
+    logo2,
+    masthead,
+    taxpayer,
+    required_links,
+    usagov
+  }) => html`
+    <usa-identifier>
+      <p slot="domain">${masthead.domain}</p>
+      ${logo1 ? html`<a slot="logo" href="${primary_agency.url}">
+        <img src="${logo1}" alt="${primary_agency.name} logo"/>
+      </a>`: null}
+      ${logo2 ? html`<a slot="logo" href="${secondary_agency.url}">
+        <img src="${logo2}" alt="${secondary_agency.name} logo"/>
+      </a>`: null}
+      <p slot="disclaimer">
+        ${masthead.disclaimer} <a href="${primary_agency.url}">${primary_agency.name}</a>
+        ${secondary_agency ? html`${masthead.conjunction} <a href="${secondary_agency.url}">${secondary_agency.name}</a>`: null}${taxpayer ? html`. ${taxpayer}`: null}
+      </p>
+      <nav slot="links">
```

Comment:
Question: 
Do we want to keep the slot on the wrapper here or should we have named slots for each individual link (e.g., `<a slot="about-link"`)?
 
I opted for adding the slot name to the wrapper here so that the user can have the flexibility to control the order of their own list and also the ability to add links to the list if they ever need to (Right now the code just loops through each child of "links", wraps it, and puts it on the page). If we want to remove that flexibility and better control the link order and link types, I can add the slot names to the links instead. Curious what you all think is better here!